src/bsa/plugin.cpp

```diff
@@ -187,13 +197,22 @@ auto list_archive(const Path &dir, const Settings &sets) noexcept -> std::vector
 {
     try
     {
-        return flux::from_range(fs::directory_iterator(dir))
+        std::vector<Path> archives = flux::from_range(fs::directory_iterator(dir))
             .filter([](const auto &f) { return f.is_regular_file(); })
             .filter([&sets](const auto &f) {
                 return common::to_lower(f.path().extension().u8string()) == sets.extension;
             })
             .map([](const auto &f) { return f.path(); })
             .to<std::vector>();
+
+        // When CAO splits archives, it adds a suffix to the path. This mean we can easily
```

Comment:
bethutil, not CAO

Reply:
Reworded the whole comment. Does it make sense?